lib/chartjs/chart_helpers.rb

```diff
@@ -36,12 +36,16 @@ def chart(klass, data, options)
       script = javascript_tag do
         <<-END.squish.html_safe
         (function() {
+        
           var initChart = function() {
             var data = #{data.to_json};
             var opts = #{options.to_json};
             if (!("animation" in opts)) {
               opts["animation"] = (typeof Modernizr == "undefined") || Modernizr.canvas;
             }
+            
+            canvas && canvas.destroy();
```

Comment:
With turbolinks, the canvas was getting this error as this issue:

https://github.com/nnnick/Chart.js/issues/899

He solved it by destroying the canvas object before creating a new one.